src/app-layout/__tests__/mobile.test.tsx

```diff
@@ -59,15 +61,24 @@ describeEachAppLayout({ sizes: ['mobile'] }, ({ theme }) => {
       : theme === 'refresh-toolbar'
         ? toolbarSkeletonStyles['toolbar-container']
         : mobileToolbarStyles['mobile-bar'];
-  const drawerBarClassName = theme.startsWith('refresh')
-    ? visualRefreshRefactoredStyles['drawers-mobile-triggers-container']
-    : drawersMobileStyles['drawers-container'];
+  const drawerBarClassName =
+    theme === 'refresh'
+      ? visualRefreshRefactoredStyles['drawers-mobile-triggers-container']
+      : theme === 'refresh-toolbar'
+        ? toolbarStyles['drawers-trigger-content']
+        : drawersMobileStyles['drawers-container'];
```

Comment:
```suggestion
const drawerBarClassName = {
   'classic': drawersMobileStyles['drawers-container'],
   'refresh': visualRefreshRefactoredStyles['drawers-mobile-triggers-container'],
   'refresh-toolbar': toolbarStyles['drawers-trigger-content']
}[theme];
```

we can also make it type-safe to ensure none of the themes are missing